test/utils.py

```diff
@@ -518,11 +518,12 @@ def is_valid_tarantool_installed(
     return True
 
 
-def get_tarantool_version():
+def get_tarantool_version(env=None):
```

Comment:
Let me suggest copying the logic from `tt-ee` to look up `tarantool` executable via the environment variable `TARANTOOL_BIN`.
This would simplify parameter passing, it would be enough to set the right variable, without additional manipulations with `PATH` and function arguments extension.
And, it will also bring the same logic to how the `tarantool` search works.

Reply:
Adjusted to use executable path instead of changing environment. `tt-ee` logic as-is is not suitable here.